doc/source/projections.rst

```diff
@@ -0,0 +1,71 @@
+.. currentmodule:: geopandas
+
+.. ipython:: python
+   :suppress:
+
+   import geopandas as gpd
+   world = gpd.GeoDataFrame().from_file('source/_example_data/naturalearth_lowres.shp')
+
+
+
+Managing Projections
+=========================================
+
+
+
+Coordinate Reference Systems
+-----------------------------
+
+CRS are important because the geometric shapes in a GeoSeries or GeoDataFrame object are simply a collection of coordinates in an arbitrary space. A CRS tells Python how those coordinates related to places on the Earth.
+
+CRS are referred to using codes called `proj4 strings <https://en.wikipedia.org/wiki/PROJ.4>`_. You can find the codes for most commonly used projections from `www.spatialreference.org <http://spatialreference.org/>`_ or `remotesensing.org <http://www.remotesensing.org/geotiff/proj_list/>`_. 
+
+The same CRS can often be referred to in many ways. For example, one of the most commonly used CRS is the WGS84 latitude-longitude projection. One `proj4` representation of this projection is: ``"+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs"``. But common projections can also be referred to by `EPSG` codes, so this same projection can also called using the `proj4` string ``"+init=epsg:4326"``. 
```

Comment:
I think of double backticks as enclosing something actually passed as code. Since I'm suggesting you can pass a string, seemed reasonable to put inside, since that's what you'd actually pass.